src/features/defi/providers/fox-farming/hooks/useFoxFarming.ts

```diff
@@ -1,6 +1,7 @@
 import { ethAssetId, fromAccountId, fromAssetId } from '@shapeshiftoss/caip'
 import { CONTRACT_INTERACTION } from '@shapeshiftoss/chain-adapters'
 import { supportsETH } from '@shapeshiftoss/hdwallet-core'
+import { getFees } from '@shapeshiftoss/utils/dist/evm'
```

Comment:
suggestion: import it from the package and not the dist, I might missed other places where we do this

Reply:
+1, probably an auto import issue.

Reply:
Agree, see comment https://github.com/shapeshift/web/pull/7249#discussion_r1663454578